src/converters/archival_object_converter.rb

```diff
@@ -68,6 +69,21 @@ def build_audit_info(object, db)
       audit_fields
     end
 
+
+    def build_dates(object, db)
+      dates = []
+
+      item = db[:item].where(:id => object[:id]).first
```

Comment:
Yep, that's a good point.  Any objects without an item will just return null. Plus, I didn't think it was a big deal if we hit the item table twice - I'll run the full migration to make sure it's not too slow!